homeassistant/components/motion_blinds/cover.py

```diff
@@ -146,6 +152,21 @@ def is_closed(self):
         """Return if the cover is closed or not."""
         return self._blind.position == 100
 
+    @callback
+    def _push_callback(self):
+        """Update entity state when a push has been received."""
+        self.schedule_update_ha_state(force_refresh=False)
```

Comment:
Don't use sync api from async context.

Reply:
does this mean I schould change it to `async_write_ha_state`?
I thought `async_write_ha_state` schould only be used inside async functions, and right now I can only register normal (sync) functions to the upstream MotionBlinds library.

However I am a bit fussy on when to use the sync/async  functions.

The upstream MotionBlinds library uses a multithread (`from threading import Thread`) to process the UDP multicast messages and the callbacks there are just regular functions (sync) that are called in the MotionBlinds library.